packages/manager/apps/dedicated/client/app/models/BillingService.class.js

```diff
@@ -46,7 +46,11 @@ export default class BillingService {
   }
 
   hasAutomaticRenew() {
-    return this.renew.automatic;
+    return (
```

Comment:
@marie-j Could you please amend your commit to apply Prettier rules (see below the expected diff)

```js
  hasAutomaticRenew() {
    return (
      this.renew.automatic ||
      this.renewalType === 'automaticV2012' ||
      this.renewalType === 'automaticV2016'
    );
  }
```